### DISCUSSION TITLE
Fonts Tutorial

### DISCUSSION THREAD

**Discussion by qwksilver** [Geektoolkit/Dynaframe3#125] [Show and tell]
Fonts work great
I recommend this:
`sudo apt install font-manager`

then download and unzip the fonts you want, from somewhere like 1001Fonts

finally rightclick the download, choose show in folder
rightclick the file, choose unzip here
rightclick the ttf, choose font manager, and click install font.

then on your settings screen use the name of the newly installed font, not case sensitive.
if you get the name wrong, or try to use a font that doesn't exist it has a default it shows instead.

**Comment by Geektoolkit** [comment]
This is excellent thankyou!
